codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/core/KotlinDependency.kt

```diff
@@ -134,6 +135,7 @@ data class KotlinDependency(
         // External third-party dependencies
         val KOTLIN_STDLIB = KotlinDependency(GradleConfiguration.Implementation, "kotlin", "org.jetbrains.kotlin", "kotlin-stdlib", KOTLIN_COMPILER_VERSION)
         val KOTLIN_TEST = KotlinDependency(GradleConfiguration.TestImplementation, "kotlin.test", "org.jetbrains.kotlin", "kotlin-test", KOTLIN_COMPILER_VERSION)
+        val KOTLINX_COROUTINES = KotlinDependency(GradleConfiguration.Implementation, "kotlinx.coroutines", "org.jetbrains.kotlinx", "kotlinx-coroutines-core", KOTLINX_COROUTINES_VERSION)
```

Comment:
Sorry, I think I gave you the wrong recommendation earlier. This could be done without hardcoding the kotlinx.coroutines version. See @0marperez PR where he does the exact same thing here: https://github.com/smithy-lang/smithy-kotlin/pull/1191/files#diff-cff225fba9b05cc2ba4b10fb157b6a2365da1a2c676cec0d06379bf14a0991ccR416

We can sync up if you have more questions